vizro-core/docs/pages/user-guides/selectors.md

```diff
@@ -26,20 +26,117 @@ For more details, kindly refer to the documentation of the underlying dash compo
 
     The later is required if you want to provide different display labels to your option values or in case you want to
     provide boolean values as options. In this case, you need to provide a string label for your boolean values as
-    boolean values cannot be displayed properly as labels in the underlying dash components.
+    boolean values cannot be displayed properly as labels in the underlying Dash components.
 
 ## Numerical selectors
 
-For more details, kindly refer to the documentation of the underlying dash components:
+For more details, refer to the documentation of the underlying Dash components:
 
-- [dcc.Slider](https://dash.plotly.com/dash-core-components/slider])
-- [dcc.RangeSlider](https://dash.plotly.com/dash-core-components/rangeslider])
+- [`dcc.Slider`](https://dash.plotly.com/dash-core-components/slider])
+- [`dcc.RangeSlider`](https://dash.plotly.com/dash-core-components/rangeslider])
 
-???+ note
+!!! note
 
     When configuring the [`Slider`][vizro.models.Slider] and the [`RangeSlider`][vizro.models.RangeSlider] with float values, and using `step` with an integer value, you may notice
     unexpected behavior, such as the drag value being outside its indicated marks.
     To our knowledge, this is a current bug in the underlying [`dcc.Slider`](https://dash.plotly.com/dash-core-components/slider) and
     [`dcc.RangeSlider`](https://dash.plotly.com/dash-core-components/rangeslider) component, which you can circumvent by adapting the `step` size accordingly.
 
-To enhance existing selectors, please see our How-to-guide on creating [custom components](custom-components.md)
+## Temporal Selectors
+
+For more details, refer to the documentation of the underlying Dash components:
+
+- [`dmc.DateRangePicker`](https://www.dash-mantine-components.com/components/datepicker#daterangepicker)
+- [`dmc.DatePicker`](https://www.dash-mantine-components.com/components/datepicker)
+
+!!! note
+
+    The [`DatePicker`][vizro.models.DatePicker] component is based on two underlying components -  `dmc.DatePicker` and `dmc.DateRangePicker`.
```

Comment:
This still renders weird for me:
![Uploading image.png…]()

Reply:
If we follow this guidance (https://github.com/mckinsey/vizro/pull/319/files#r1528241690), then we could also shorting this note significantly. I would then just add this explanation below the bullet point about this selector.

The note should then contain the only in my eyes important thing for the reader to note, which is the need to provide the correct date formats.

Given that this section is before the other, I wonder still how to make it clear that `datetime` actually is

Reply:
Yes, that looks ok! Although my comment about shorting this still holds :)